settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
If I do that, this happens:

![image](https://cloud.githubusercontent.com/assets/1140238/15112795/6e55af38-15e8-11e6-9dcb-dcf8a5db896a.png)